narwhal/executor/tests/consensus_integration_tests.rs

```diff
@@ -19,7 +19,7 @@ use types::{Certificate, ReconfigureNotification, TransactionProto};
 
 #[tokio::test]
 async fn test_recovery() {
-    let _guard = setup_tracing();
+    // let _guard = setup_tracing();
```

Comment:
dead code?

Reply:
Ah just left this here so in case of debugging someone can easily uncomment this and see the logs. But can totally remove it.